optlib/inko.ctags

```diff
@@ -0,0 +1,79 @@
+# inko.ctags --- regex parser for the Inko programming language
+# (https://inko-lang.org/)
+#
+#  Copyright (c) 2019, Yorick Peterse
+#
+#  Author: Yorick Peterse <yorickpeterse@gmail.com>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,
+# USA.
+
+--langdef=Inko
+--map-Inko=+.inko
+--kinddef-Inko=o,object,Object definition
+--kinddef-Inko=m,method,Method definition
+--kinddef-Inko=t,trait,Trait definition
+--kinddef-Inko=a,attribute,Attribute definition
+--kinddef-Inko=c,constant,Constant definition
+--kinddef-Inko=r,reopen,Reopen object
+
+--_fielddef-Inko=implements,Trait being implemented
+--_fielddef-Inko=signature,Method signature
```

Comment:
It seems that this signature field is not used anywhere.
If you may have a plan to implement it. you can add this definition in the future.